src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsDetailsMapping.java

```diff
@@ -252,8 +255,7 @@ public static boolean shouldBeHearingsInWelshFlag() {
     }
 
     public static List<HearingLocation> getHearingLocations(SscsCaseData caseData,
-                                                            ReferenceDataServiceHolder referenceDataServiceHolder) {
-
+                                                            ReferenceDataServiceHolder referenceDataServiceHolder) throws URISyntaxException, IOException {
         OverrideFields overrideFields = OverridesMapping.getOverrideFields(caseData);
 
         if (isNotEmpty(overrideFields.getHearingVenueEpimsIds())) {
```

Comment:
Do we need to consider this case? If an override exists we might still want to send the extra venues - one to check with the business

Reply:
This code was added when I rebased it to master, looks like this is the logic for another ticket. Would probably be safe to leave it in there.

Reply:
I'd ask the business whether they want override fields to override the extra epims ID

Reply:
Asked Makeda to confirm this from Po's

Reply:
Response from Makeda :
 When using the override then 1 venue is correct as the user will be specific about the location that they want to list into. as confirmed with POs  Sheila and Rosie